azure-pipelines.yml

```diff
@@ -23,3 +23,58 @@ jobs:
   parameters:
     name: WindowsPy27
     vmImage: vs2017-win2017
+
+- job: 'Linting'
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.7'
+      architecture: 'x64'
+  # We're ignoring the following codes across the board
+  #E402,  # module level import not at top of file
+  #E731,  # do not assign a lambda expression, use a def
+  #E741,  # do not use variables named 'l', 'O', or 'I'
+  #W503,  # line break before binary operator
+  #C406,  # Unnecessary (list/tuple) literal - rewrite as a dict literal.
+  #C408,  # Unnecessary (dict/list/tuple) call - rewrite as a literal.
+  #C409,  # Unnecessary (list/tuple) passed to tuple() - (remove the outer call to tuple()/rewrite as a tuple literal).
+  #C410   # Unnecessary (list/tuple) passed to list() - (remove the outer call to list()/rewrite as a list literal).
+
+  # pandas/_libs/src is C code, so no need to search there.
+  - script: flake8 pandas --filename=*.py --exclude pandas/_libs/src --ignore=C406,C408,C409,E402,E731,E741,W503
```

Comment:
Is this `--ignore` necessary? I would think that this would be picked up from our setup.cfg. Better to delete it here so that we don't have to update two places.

Reply:
That's a good point. That was in the original `lint.sh`, but will check if the setting in `setup.cfg` is used.